examples/template/.vscode/launch.json

```diff
@@ -1,39 +1,47 @@
 {
 	"configurations": [
 		{
-			"name": "GDB Debug Target",
+			"name": "cppdbg GDB Debug Target",
 			"type": "cppdbg",
 			"request": "launch",
-			"program": "template.elf",
+			"program": "${workspaceFolder}/template.elf",
 			"args": [],
 			"stopAtEntry": true,
 			"cwd": "${workspaceFolder}",
 			"environment": [],
 			"externalConsole": false,
 			"MIMode": "gdb",
-			"deploySteps": [
+			"deploySteps": [ // 'make ...gdbserver' doesn't seem to work here. The Makefile calls 'minichlink -baG'... Needs -aG. Easier to add minichlink as a seperate step below
+				{
+					"type": "shell", //isn't there some way to call a task from tasks.json?
+					"command": "make --directory=${workspaceFolder} -j 1 closechlink flash",
+				},
 				{
 					"type": "shell",
-					"continueOn": "GDBServer",
-					"command": "make --directory=${workspaceFolder} closechlink flash gdbserver"
+					"command": "${workspaceFolder}/minichlink/minichlink -aG",
```

Comment:
So the above change in `program` as `${workspaceFolder}/template.elf` indicates that `${workspaceFolder}` is pointing to the actual example folder, not the `ch32v003fun` "root" directory itself. Then how can minichlink exist at `${workspaceFolder}/minichlink/minichlink -aG` when you rewrote this command?

Maybe these template files have been written with the expectation that they be copied into the root of the ch32v003fun folder?

Reply:
oh, yes, you are correct. The paths are still set to my own project directory structure, not the 'fun examples structure. Will update when I get a sec

Reply:
Thanks, will recheck project in VSCode to make sure everything's fine.